cartoframes/context.py

```diff
@@ -218,6 +218,8 @@ def read(self, table_name, limit=None, decode_geom=False, shared_user=None):
               <http://geopandas.org/>`__.
             shared_user (str, optional): If a table has been shared with you,
               specify the user name (schema) who shared it.
+            retry_times (str, optional): If the read call is rate limited,
```

Comment:
Should this be an `int` instead of `str`?

Reply:
:+1:

Reply:
I will use this to talk about
> why not re-use CartoContext.query from CartoContext.read since there are so many similarities in how the data is structured and returned?

I dont want to use for 2 reasons:
- first of all, query can receive not only SELECT queries (COPY TO), but INSERT (COPY FROM), UPDATE (Batch API), DELETE, ...
- we are going to "make stupid" the `query` method. We are still talking about, but the idea is to ask the user is the `query` is for reading or for writing to use the appropiate API

Reply:
Cool, that all makes sense